qrenderdoc/Windows/PipelineState/VulkanPipelineStateViewer.cpp

```diff
@@ -41,6 +41,13 @@ Q_DECLARE_METATYPE(CombinedSamplerData);
 
 namespace
 {
+
+template <typename T>
+T RDCMIN(const T &a, const T &b)
+{
+  return a < b ? a : b;
+}
+
```

Comment:
Please don't define new macros in qrenderdoc like this. The Qt code uses `qMin` and `qMax` as defined by Qt to fit that project.